stdlib/src/builtin/_math.mojo

```diff
@@ -267,3 +267,71 @@ fn gcd(*values: Int) -> Int:
         if result == 1:
             return result
     return result
+
+
+# ===----------------------------------------------------------------------=== #
+# lcm
+# ===----------------------------------------------------------------------=== #
+
+
+fn lcm(owned m: Int, owned n: Int, /) -> Int:
+    """Computes the least common multiple of two integers.
+
+    Args:
+        m: The first integer.
+        n: The second integer.
+
+    Returns:
+        The least common multiple of the two integers.
+    """
+    return abs((m // gcd(m, n)) * n) if m > n else abs((n // gcd(m, n)) * m)
```

Comment:
Probably need to check for zero divisor. Please also add a test.
```suggestion
    if d: = gcd(m, n):
        return abs((m // d) * n if m > n else (n // d) * m)
    return 0 
```

Reply:
ooooo walrus operator